taskcluster/fenix_taskgraph/transforms/browsertime.py

```diff
@@ -123,3 +123,22 @@ def fill_email_data(config, tasks):
             email["subject"] = email["subject"].format(**format_kwargs)
 
         yield task
+
+
+@transforms.add
+def filter_ytp_tests(config, tasks):
+    """Youtube-playback AV1, H264 and VP9 tests should run on P2 only"""
+    for task in tasks:
+        test_platform = task['test-platform']
+        test_name = task["test-name"]
+        if test_platform.startswith('android-hw-p2'):
```

Comment:
With this transform we will only schedule AV1, H264, and VP9 tests on P2. All the other tests will be skipped.

You will need to let all tests for P2 through, but restrict the tests that can run on the G5 (all but VP9 and AV1 tests).

Reply:
In the comments above you said that AV1, H264, and VP9 tests should only run on P2. Did something changed?
If the device is not P2, it won't get into the `if` but it will be yielded by the last line of the function.

Reply:
Nothing changed - you misunderstood the comment. Those test variants should only run on P2 and not the G5. What you implemented here is "P2 should only run these X tests" rather than "these X tests should only run on P2"